devito/equation.py

```diff
@@ -168,5 +174,6 @@ def solve(eq, target, **kwargs):
     # turnaround time
     kwargs['rational'] = False  # Avoid float indices
```

Comment:
shoudn't what comes from `kwargs` be initialized somewhere before, so we avoid using it like an actual parameter?